src/main/java/interface_adapter/club_logged_in/club_create_post/ClubCreatePostController.java

```diff
@@ -23,4 +23,18 @@ public void execute(String email, String title, String content) {
         final ClubCreatePostInputData createPostInputData = new ClubCreatePostInputData(email, title, content);
         createPostInteractor.execute(createPostInputData);
     }
+
+    /**
+     * Executes the Switch to Create Post View Use Case.
+     */
+    public void switchToCreatePostView() {
+        createPostInteractor.switchToCreatePostView();
+    }
+
+    /**
+     * Executes the Switch to Club Logged In View Use Case.
+     */
+    public void switchToClubLoggedInView() {
+        createPostInteractor.switchToClubLoggedInView();
```

Comment:
I see. I thought we could do it in the view code, but if this was better for you I see no problem as it is still CA!

Reply:
Yes I thought about it a little more and decided to do this approach since it's what we did for other cases. It also helps prevent clutter in the Views from more imports